src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -202,32 +196,81 @@ namespace winrt::TerminalApp::implementation
 
             e.Handled(true);
         }
+        else
+        {
+            const auto vkey = ::gsl::narrow_cast<WORD>(e.OriginalKey());
+
+            // In the interest of not telling all modes to check for keybindings, limit to TabSwitch mode for now.
+            if (_currentMode == CommandPaletteMode::TabSwitchMode)
+            {
+                auto const ctrlDown = WI_IsFlagSet(CoreWindow::GetForCurrentThread().GetKeyState(winrt::Windows::System::VirtualKey::Control), CoreVirtualKeyStates::Down);
+                auto const altDown = WI_IsFlagSet(CoreWindow::GetForCurrentThread().GetKeyState(winrt::Windows::System::VirtualKey::Menu), CoreVirtualKeyStates::Down);
+                auto const shiftDown = WI_IsFlagSet(CoreWindow::GetForCurrentThread().GetKeyState(winrt::Windows::System::VirtualKey::Shift), CoreVirtualKeyStates::Down);
+
+                auto success = _bindings.TryKeyChord({
+                    ctrlDown,
+                    altDown,
+                    shiftDown,
+                    vkey,
+                });
```

Comment:
Hmm... This is a bit concerning. Doesn't this open up the possibility of something like this?
- <kbd>ctrl+c</kbd> is bound to "Copy"
- I press <kbd>ctrl+c</kbd> when trying to copy something from the search box
- Copy Action is invoked on the focused terminal below

We're using this to just find `nextTab` and `prevTab` right? Should we only include those two actions in `_bindings`?

Reply:
I honestly don't know how to trim down the given bindings to just `nextTab` and `prevTab` 😢 

It's something I was _definitely_ thinking about. I'm mostly hoping that because the control needs to be kept open with a modifier key, and there's no search bar and the only interactable interface is a list view, the range of actions a user would want to do is limited. 

Like, if you've hit ctrl+tab to open the tab switcher, and for some ODD reason you shift your fingers to invoke `find` which could be something like ctrl+shift+f, it'll show the find dialog and leave the tab switcher open. But, that's like going out of the way to break things especially since you're deliberately anchoring the UI in place while attempting to open something else.